src/mono/wasm/templates/templates/browser/Program.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Runtime.CompilerServices;
+
+Console.WriteLine ("Hello, Console!");
+
+public class MyClass {
+    [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
Do we still need this `NoInlining` ?

Reply:
I guess its purpose is to avoid inlining the method during AOT. Not sure whether anything changed recently in that area though. @lewing, do you know if we need it?